idea/src/org/jetbrains/kotlin/idea/quickfix/AddExclExclCallFix.kt

```diff
@@ -50,42 +38,8 @@ import org.jetbrains.kotlin.util.OperatorNameConventions
 import org.jetbrains.kotlin.util.isValidOperator
 import org.jetbrains.kotlin.utils.addToStdlib.safeAs
 
-
-abstract class ExclExclCallFix(psiElement: PsiElement) : KotlinQuickFixAction<PsiElement>(psiElement) {
-    override fun getFamilyName(): String = text
-
-    override fun startInWriteAction(): Boolean = true
-}
-
-class RemoveExclExclCallFix(psiElement: PsiElement) : ExclExclCallFix(psiElement), CleanupFix, HighPriorityAction {
-    override fun getText(): String = KotlinBundle.message("fix.remove.non.null.assertion")
-
-    override fun isAvailable(project: Project, editor: Editor?, file: KtFile): Boolean =
-        getExclExclPostfixExpression() != null
-
-    override fun invoke(project: Project, editor: Editor?, file: KtFile) {
-        if (!FileModificationService.getInstance().prepareFileForWrite(file)) return
-
-        val postfixExpression = getExclExclPostfixExpression() ?: return
-        val expression = KtPsiFactory(project).createExpression(postfixExpression.baseExpression!!.text)
-        postfixExpression.replace(expression)
-    }
-
-    private fun getExclExclPostfixExpression(): KtPostfixExpression? {
```

Comment:
If the above `isAvailable` is restored, I guess this one can be improved and reused in `QuickFixesPsiBasedFactory#doCreateQuickFix`?

Reply:
That's pretty much what I already did. Instead of finding the parent `KtPostfixExpression` during invocation, we just do it when creating the quick fix in `QuickFixesPsiBasedFactory#doCreateQuickFix`.